packages/legend-application-studio/src/components/editor/edit-panel/mapping-editor/MappingTestEditor.tsx

```diff
@@ -307,6 +355,14 @@ const MappingTestQueryEditor = observer(
             changeClassMapping={changeClassMapping}
           />
         )}
+        {editorStore.embeddedQueryBuilderState.queryBuilderState
+          ?.textEditorState.mode && (
+          <QueryBuilderTextEditor
```

Comment:
Why is this even needed. 
If you set the edit mode to `TEXT`. Then ` <EmbeddedQueryBuilder />` shoudl handle the correct rendering of it